frontend/model/contracts/group.js

```diff
@@ -616,6 +638,10 @@ sbp('chelonia/defineContract', {
           } else {
             updateCurrentDistribution({ contractID, meta, state, getters })
           }
+
+          // TODO: when 'PAYMENT_CANCELED' is completely implemented,
+          //       deduce the amount from 'totalPledgeAmount'.
```

Comment:
I think you're thinking of is `PAYMENT_REVERSED`, because `PAYMENT_CANCELLED` can't happen after `PAYMENT_COMPLETED`. Currently, we don't support `PAYMENT_REVERSED` at all, so could you modify this to say the following?

```js
// NOTE: if 'PAYMENT_REVERSED' is implemented, subtract
//       the amount from 'totalPledgeAmount'.
```